scout/server/blueprints/institutes/controllers.py

```diff
@@ -131,6 +132,45 @@ def institute(store, institute_id):
     return data
 
 
+def populate_beacon_form(institute_obj):
+    """Populate the form select of scout.server.blueprints.institutes.forms.BeaconDatasetForm with data.
+    Available data is a controlled dictionary of dataset names depending of the datasets already existing in the Beacon server
+
+    Args:
+        institute_obj(dict) An institute object
+    """
+    beacon_form = BeaconDatasetForm()
+
+    if current_user.is_admin is False:
+        LOG.warning("User must be an admin to create a beacon dataset")
```

Comment:
That will be many warnings; most users are not admin. Better at the actual action endpoint!